### PR TITLE
[Feature] Added SetWindowOpacity(float opacity)

### PR DESCRIPTION
**Pull request by tusharsingh09**
`SetWindowOpacity()` takes `opacity`(`float`) as a parameter. Value of `opacity` must fall under 0 and 1. If it exceeds 1; it will be equal 1, similarly if it's smaller than 0; opacity is equal to 0.
Random Screenshots (*READ THE VALUE INSIDE SetWindowOpacity()*):
set_window_opacity_0
set_window_opacity_1
set_window_opacity_2
set_window_opacity_3
 `SetWindowOpacity(99999999.0)` goes same as `SetWindowOpacity(1.0)`.

Thank you

### CODE REVIEW

File: src/rcore.c [hunk 1]
```diff
@@ -1534,6 +1534,16 @@ void SetWindowSize(int width, int height)
 #endif
 }
 
+// Set window opacity, value opacity is between 0.0 and 1.0
+void SetWindowOpacity (float opacity)
```
Comment (raysan5) [inline src/rcore.c:1538]: It seems there is an extra space before parenthesis, please review it.

File: src/rcore.c [hunk 2]
```diff
@@ -1534,6 +1534,16 @@ void SetWindowSize(int width, int height)
...
+{
+#if defined(PLATFORM_DESKTOP)
+    if (opacity >= 1.0) opacity = 1.0;
```
Comment (raysan5) [inline src/rcore.c:1541]: float values should end with `f` --> `1.0f`, `0.0f`

**Comment by raysan5** [comment]
@tusharsingh09 This is a nice addition, added a couple of comments for review.

File: src/rcore.c (hunk 1 above)
Comment (tusharsingh09) [reply src/rcore.c:1538]: I'm really sorry, it was intentional I thought I read it in CONTRIBUTING.md but anyways I'll fix it.

File: src/rcore.c (hunk 2 above)
Comment (tusharsingh09) [reply src/rcore.c:1541]: Yup, okay

**Comment by raysan5** [comment]
@tusharsingh09 thanks for the addition!

**Pull request merged**